src/main/frontend/handler/events.cljs

```diff
@@ -89,10 +89,7 @@
 (defn- enable-beta-features!
   []
   (when-not (false? (state/enable-sync?)) ; user turns it off
-    (file-sync-handler/set-sync-enabled! true))
-
-  (when-not (false? (state/enable-whiteboards?))
-    (config-handler/set-config! :feature/enable-whiteboards? true)))
```

Comment:
:feature/enable-whiteboards? the default value is true now. so these lines are redundant